{
  "type": "object",
  "properties": {
    "owner": {
      "description": "The account owner of the repository. The name is not case sensitive.",
      "type": "string"
    },
    "repo": {
      "description": "The name of the repository without the `.git` extension. The name is not case sensitive.",
      "type": "string"
    },
    "pull_number": {
      "description": "The number that identifies the pull request.",
      "type": "integer"
    },
    "review_id": {
      "description": "The unique identifier of the review.",
      "type": "integer"
    },
    "body": {
      "type": "string",
      "description": "The body text of the pull request review"
    },
    "event": {
      "type": "string",
      "description": "The review action you want to perform. The review actions include: `APPROVE`, `REQUEST_CHANGES`, or `COMMENT`. When you leave this blank, the API returns _HTTP 422 (Unrecognizable entity)_ and sets the review action state to `PENDING`, which means you will need to re-submit the pull request review using a review action.",
      "enum": [
        "APPROVE",
        "REQUEST_CHANGES",
        "COMMENT"
      ]
    }
  },
  "required": [
    "owner",
    "repo",
    "pull_number",
    "review_id",
    "event"
  ]
}